src/joint_state_listener.cpp

```diff
@@ -133,8 +137,8 @@ void JointStateListener::callbackJointState(const JointStateConstPtr& state)
   // check if we need to publish
   if (ignore_timestamp_ || state->header.stamp >= last_published + publish_interval_) {
     // get joint positions from state message
-    map<string, double> joint_positions;
-    for (unsigned int i=0; i<state->name.size(); i++) {
+    std::map<std::string, double> joint_positions;
+    for (unsigned int i = 0; i < state->name.size(); i++) {
```

Comment:
Same here; make i a `size_t`.

Reply:
size_t in d780506c444f1519c78c29717b7f4296a884b016